### ISSUE TITLE
No automatic syntax highlighting for code blocks

### ISSUE THREAD

**Issue by inspiraller** [GaelGirodon/markdown-to-document#5]
**When I run this command:**
mdtodoc markdownfiles/*.md --dest dist --theme github --highlight-style atom-one-light,

I get no syntax highlighting for my code blocks
I would expect this to have syntax colouring for my html and javascript as below.

Example.md
# Some js file stuf here 
```js
const x = "hello"
console.log(x)
```

# Some html stuff here...
```html
<!doctype html>
<html lang="en">
<body>
  <div id="app">Placeholder.</div>

  <script>
    // do something here...
    document.getElementById('app').innerHTML = 'hello world';
  </script>
</body>
</html>
```

**Comment by inspiraller** [comment]
I've temporarily resolved it by downloading the highlight.js files from this cdn: https://highlightjs.org/
into my local directory.

Then using your custom template 
mdtodoc markdownfiles/*.md --dest dist --theme github --layout layout/custom-layout.html"

**custom-layout.html***
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <!--        ⬐ Markdown document title included here -->
  <title>{{ title }}</title>
  {{ styles }} <!-- ← CSS styles (theme, highlight styles, etc.) included here -->


  <!-- syntax highlighting for code blocks -->
  <link rel="stylesheet" href="../lib/highlight/highlight.min.css">
  <script src="../lib/highlight/highlight.min.js"></script>

  
  <link rel="stylesheet" href="../css/global.css" />
</head>
<body class="highlight">
{{ body }}     <!-- ← Compiled Markdown included here -->
</body>
{{ scripts }}  <!-- ← JS scripts included here -->
</html>
```

**Comment by GaelGirodon** [comment]
**TL;DR**: add `--layout basic` to your first command

You're on the right track: you need to provide a layout to "host" styles and scripts tags required for syntax highlighting.
You don't necessarily need to write it by yourself as some basic ones are built-in (`basic` and `page`):

```diff
- mdtodoc example.md -d dist -t github -s atom-one-light
+ mdtodoc example.md -d dist -t github -s atom-one-light -l basic
```

When you omit the layout, you get a "raw" compilation of your Markdown. If you look at the output HTML source file, there is only the body content and so, syntax highlighting won't work because associated styles and scripts were not included.
